File: python/ql/test/experimental/dataflow/import-helper/test5.py
import mypkg

print(mypkg.foo)  # 42
try:
    print(mypkg.bar)
except AttributeError as e:
    print(e)  # module 'mypkg' has no attribute 'bar'

from mypkg import bar as _bar
print(mypkg.bar)  # <module 'mypkg.bar' ...

File: java/ql/test/query-tests/InconsistentOperations/Test5.java
public class Test5 {
	static class A { void bar() {} }
	static A foo() { return new A(); }

	void test() {
		{ A a = foo(); a.bar(); }
		{ A a = foo(); a.bar(); }
		{ A a = foo(); a.bar(); }
		{ A a = foo(); a.bar(); }
		{ A a = foo(); a.bar(); }
		{ A a = foo(); a.bar(); }
		{ A a = foo(); a.bar(); }
		{ A a = foo(); a.bar(); }
		{ A a = foo(); a.bar(); }
	}

	static A a = foo(); // OK
}